### PR TITLE
docs: update header to new design

### PR DESCRIPTION
**Pull request by seve**
image
 #835 
 
 Note: There was an untracked directory added at some point during my doc HTML generation (`_autosummary/`). Should I commit, add to .gitignore, or no action?

### CODE REVIEW

File: docs/_static/css/custom.css
Comment (seve) [inline docs/_static/css/custom.css:1]: prettier (the team standard formatter for .css) got to the file, should I revert?

File: docs/_static/css/custom.css [hunk 1]
```diff
@@ -1,307 +1,538 @@
...
+  letter-spacing: 0.33px;
+  font-weight: 600;
+  text-transform: uppercase;
+  color: #767676;
+  font-weight: 700;
+}
+
+.navbar-cxg-nav-item-container {
+  display: flex;
+  gap: 16px;
```
Comment (seve) [inline docs/_static/css/custom.css:73]: Actual code changes

File: docs/_static/css/custom.css
Comment (ebezzi) [reply docs/_static/css/custom.css:1]: I don't mind.

**Review by ebezzi — APPROVED** [review]

**Review by tihuan — APPROVED** [review]
LGTM! Thanks for syncing census with the latest navbar, @seve !

**Pull request merged**